snippets/legv8.json

```diff
@@ -0,0 +1,208 @@
+{
+  "set register": {
+    "prefix": "sreg",
+    "body": ["ADD ${1:Rd}, XZR, ${2:Rm}", "$3"],
+    "description": "R[1] = R[2]"
+  },
+  "increment register": {
+    "prefix": "ireg",
+    "body": ["ADDI ${1:Rd}, ${1}, #${2:Imt}", "$3"],
+    "description": "R[1] = R[1] + R[2]"
+  },
+  "decrement register": {
+    "prefix": "dreg",
+    "body": ["SUBI ${1:Rd}, ${1}, #${2:Imt}", "$3"],
+    "description": "R[1] = R[1] - R[2]"
+  },
+
+  "stack manipulation with 2 registers": {
```

Comment:
I was thinking the same thing, but like you said I was stuck on the technicality of it since LR is also a register. But I think that being more intuitive is more important!